multiversx_sdk_core/transaction_builders/esdt_builders.py

```diff
@@ -70,6 +71,6 @@ def _build_payload_parts(self) -> List[str]:
             *(args_to_strings(["canMint", "true"]) if self.can_mint else []),
             *(args_to_strings(["canBurn", "true"]) if self.can_burn else []),
             *(args_to_strings(["canChangeOwner", "true"]) if self.can_change_owner else []),
-            *(args_to_strings(["canUpgrade", "true"]) if self.can_upgrade else []),
-            *(args_to_strings(["canAddSpecialRoles", "true"]) if self.can_add_special_roles else [])
+            *(args_to_strings(["canUpgrade", str(self.can_upgrade).lower()])),
```

Comment:
I've sent a transaction explicitly setting `canUpgrade` to `False` and it works. I don't think that the constructor needs to be adjusted because we don't want any change since this will be deprecated in the next release.